src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/help-excludedRegions.html

```diff
@@ -1,14 +1,15 @@
 <div>
     Each exclusion uses regular expression pattern matching, and must be separated by a new line.
-    <p/>
-  <pre>
-	myapp/src/main/web/.*\.html
-	myapp/src/main/web/.*\.jpeg
-	myapp/src/main/web/.*\.gif
-  </pre>
+    <br/>
+    <pre>
+	    myapp/src/main/web/.*\.html
```

Comment:
You're mixing tabs and spaces here.  Please fix all instances of using tabs and switch them to using 4 spaces per tab.  I'm only going to comment this once but this feedback applies to all changes in this PR which contain tabs.